2-dom-manipulation/form.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="es">
```

Comment:
```suggestion
<html lang="en">
```

Reply:
You have some labels in Spanish and some in English 🤔

Reply:
I always forget it.